src/tools/tabs.ts

```diff
@@ -50,9 +50,9 @@ const selectTab: ToolFactory = captureSnapshot => defineTool({
   schema: {
     name: 'browser_tab_select',
     title: 'Select a tab',
-    description: 'Select a tab by index',
+    description: 'Select a tab by index (0-based indexing)',
```

Comment:
Please revert all the description changes - 0-based seems to be the default for the agent, so there's no need to make it explicit.